go.sum

```diff
@@ -78,6 +84,8 @@ github.com/ONSdigital/dp-net/v2 v2.3.0 h1:Qm4OmtLRS5RDVk/1uC6NvKJVsTJL+nLUWWjl4S
 github.com/ONSdigital/dp-net/v2 v2.3.0/go.mod h1:8X7E1wiJxL59qaDFt3ShDfFLfIGs4WMRPx1sZIgAqbU=
 github.com/ONSdigital/dp-renderer v1.20.1 h1:7iz25P+Dc01O5QSr6jkMpmamU5wdUBnpGzujpRGdNZE=
 github.com/ONSdigital/dp-renderer v1.20.1/go.mod h1:Z5sbyS0ApY6D8ikGDt2KIl5bt2p5kk73PeeMxGs1pEM=
+github.com/ONSdigital/dp-renderer v1.28.0 h1:BInquwPyselv6PnkOWWwAqbabqm0XJWbPlhMwzP+wCs=
+github.com/ONSdigital/dp-renderer v1.28.0/go.mod h1:Z5sbyS0ApY6D8ikGDt2KIl5bt2p5kk73PeeMxGs1pEM=
```

Comment:
Would benefit from a `go mod tidy` to prune old dependencies

Reply:
Fair comment, as this is a prod release candidate I'll do the commit following the usual process of PR into dev then it will make it's way into prod in the near future. I don't think your request warrants hot fixing this branch.